content/departments/engineering/product/process/index.md

```diff
@@ -38,6 +38,30 @@ See [strategy](../../../../strategy-goals/index.md).
 - [Pricing and tiering/packaging](gtm/pricing.md)
 - [Licensing](gtm/licensing.md)
 
+#### Experimenting on sourcegraph.com
+
+We welcome experimentations on sourcegraph.com. It should be very easy to experiment on .com, send a tweet or email campaign out to get some traffics, and get early feedback in.
+
+The bar for an experiment on .com is the following:
+
+- Break nothing (new UIs and public data APIs are low risk, anything involving sensitive data or critical APIs is high risk)
+- Don't disrupt the main flows
+- Add basic usage metrics to know what's going on
+
+In particular an experiment does not require:
+
+- Discoverability
+- A polished design
+- Approvals, as long as it meets the criteria above
+- A/B testing, although A/B testing can be used
+
+When you experiment on .com you commit to:
```

Comment:
Should we include something about announcing it on some internal channel to encourage internal feedback. If so, which one might be best for announcing it? #progress ?

Reply:
#progress sounds good!